docs/capabilities/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Capability Catalog'
+---
+
+**Capabilities** are pluggable pieces of functionality for your applications. Plug them in and swap them out whenever you want.
+
+Every capability consists of a **standard API** and one or more swappable plugins called **capability providers**. Learn more about capabilities, providers, interfaces, and other core concepts in the [Platform Guide](/docs/concepts/index.mdx).
```

Comment:
Do we want to mention WIT here? Maybe it's too much of a sidetrack this early? 

Also, what do you think about standard API "provided by" Capability Providers (and then capability providers being a link to that doc)

Are we just using the word "provider" instead of "capability provider" these days?